### PR TITLE
Do not interrupt kinetic scroll by single arrow animation.

### PR DESCRIPTION
**Pull request by darina**

### CODE REVIEW

File: drape_frontend/animation/animation.cpp [hunk 1]
```diff
@@ -90,4 +90,45 @@ bool Animation::GetMaxDuration(Interpolator const & interpolator, double & maxDu
   return true;
 }
 
+std::string DebugPrint(Animation::Type const & type)
+{
+  if (type == Animation::Type::Sequence)
```
Comment (ygorshenin) [inline drape_frontend/animation/animation.cpp]: Please, use switch-case instead.

File: drape_frontend/animation/animation.cpp [hunk 2]
```diff
@@ -90,4 +90,45 @@ bool Animation::GetMaxDuration(Interpolator const & interpolator, double & maxDu
...
+  if (type == Animation::Type::Arrow)
+    return "Arrow";
+  if (type == Animation::Type::KineticScroll)
+    return "KineticScroll";
+  return "Unknown type";
+}
+
+std::string DebugPrint(Animation::Object const & object)
+{
+  if (object == Animation::Object::MyPositionArrow)
```
Comment (ygorshenin) [inline drape_frontend/animation/animation.cpp]: Please, use switch-case instead.

File: drape_frontend/animation/animation.cpp [hunk 3]
```diff
@@ -90,4 +90,45 @@ bool Animation::GetMaxDuration(Interpolator const & interpolator, double & maxDu
...
+  if (object == Animation::Object::MapPlane)
+    return "MapPlane";
+  if (object == Animation::Object::Selection)
+    return "Selection";
+  return "Unknown object";
+}
+
+std::string DebugPrint(Animation::ObjectProperty const & property)
+{
+  if (property == Animation::ObjectProperty::Position)
```
Comment (ygorshenin) [inline drape_frontend/animation/animation.cpp]: Please, use switch-case instead.

Comment (darina) [inline drape_frontend/animation/animation.cpp]: done

File: drape_frontend/animation/animation.cpp [hunk 4]
```diff
@@ -90,4 +90,41 @@ bool Animation::GetMaxDuration(Interpolator const & interpolator, double & maxDu
...
+  switch (type)
+  {
+  case Animation::Type::Sequence: return "Sequence";
+  case Animation::Type::Parallel: return "Parallel";
+  case Animation::Type::MapLinear: return "MapLinear";
+  case Animation::Type::MapScale: return "MapScale";
+  case Animation::Type::MapFollow: return "MapFollow";
+  case Animation::Type::Arrow: return "Arrow";
+  case Animation::Type::KineticScroll: return "KineticScroll";
+  default: return "Unknown type";
```
Comment (ygorshenin) [inline drape_frontend/animation/animation.cpp]: Please, don't use default unless you're really want to handle all possible cases. Just
```return "Unknown type";``` at the end of the function.

Comment (darina) [inline drape_frontend/animation/animation.cpp]: done

**Pull request merged**